docs/Other SDL Features/Encryption/index.md

```diff
@@ -79,11 +79,14 @@ ServiceEncryptionListener serviceEncryptionListener = new ServiceEncryptionListe
 !@
 
 ## Setting Optional Encryption
-If you want to encrypt a specific RPC, you must configure the payload protected status of the RPC before you send it to the head unit.
+If you want to encrypt a specific RPC, you must configure the payload protected status of the RPC before you send it to the head unit. Before sending RPCs with encryption you should call `startRPCEncryption` on the `sdlManager`.  The best place to put `startRPCEncryption` is in the successful callback of  `startWithReadyHandler`.
```

Comment:
Can you wrap this new sentence in iOS platform tags and add another platform tag for java-suite with a `TODO` comment. They have a different way of doing things.